packages/hs-react-ui/src/components/Button/ButtonContainers.tsx

```diff
@@ -36,9 +125,41 @@ export const DestructiveButtonContainer = styled(ButtonContainer)`
   }
 `;
 
-export const ButtonContainers = {
+export const LinkButtonContainer = styled(ButtonContainer)`
+  ${({ color }: StyledButtonProps) => `
+  color: ${color || colors.grayDark};
+  background-color: ${colors.transparent};
+  &:hover {
+    background-color: ${colors.transparent};
+  }
+  &:active {
+    background-color: ${colors.transparent};
+  }
+  `}
+`;
+
+export const OutlineButtonContainer = styled(ButtonContainer)`
+  ${({ color, fillColor }: StyledButtonProps) => `
+  color: ${color || colors.grayDark};
+  background-color: ${fillColor ||
+    getComplementaryColor(color || colors.grayDark, colors.background)};
+  border: 1px solid ${color || colors.grayDark};
+  &:hover {
+    background-color: ${darken(0.05, fillColor || getComplementaryColor(color || colors.grayDark, colors.background))};
+  }
+  &:active {
+    background-color: ${darken(0.1, fillColor || getComplementaryColor(color || colors.grayDark, colors.background))};
+  }
+  `}
+`;
+
+export const ButtonContainers: {
+  [index: string]: StyledComponentBase<any, {}, StyledButtonProps>;
+} = {
   [ButtonTypes.default]: ButtonContainer,
   [ButtonTypes.primary]: PrimaryButtonContainer,
   [ButtonTypes.secondary]: SecondaryButtonContainer,
   [ButtonTypes.destructive]: DestructiveButtonContainer,
+  [ButtonTypes.link]: LinkButtonContainer,
+  [ButtonTypes.outline]: OutlineButtonContainer,
 };
```

Comment:
Can we break from this pattern? I'd prefer just one container which can be easily overwritten - but which receives all the data for type and color as props

If that doesn't sound right lemme know your thoughts